Issue starting project, typo on filename

## Bug Report

**Current Behavior**
`yarn start` outputs the following error:

```stdout
yarn run v1.22.4
$ node code.js
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '/app/code.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Possible Solution**
https://github.com/LucasCtrl/spookyBot/blob/703622101188836e0ed6d11ca445e5eeddc2b373/package.json#L23

This should be `core.js` instead of `code.js`

--- a/package.json
+++ b/package.json
@@ -20,6 +20,6 @@
   },
   "scripts": {
     "dev": "nodemon --inspect core.js",
-    "start": "node code.js"
+    "start": "node core.js"
   }
 }

